components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/DBDeployer.java

```diff
@@ -294,16 +294,18 @@ private boolean isFaultyService(DeploymentFileData deploymentFileData) {
     }
 
     private String getServiceNameFromDSContents(File file) throws Exception {
-        StAXOMBuilder builder = new StAXOMBuilder(new FileInputStream(file.getAbsoluteFile()));
-        OMElement serviceEl = builder.getDocumentElement();
-        String serviceName = serviceEl.getAttributeValue(new QName(DBSFields.NAME));
-        builder.close();
-        if (DBUtils.isEmptyString(serviceName)) {
-            throw new DataServiceFault("Service group cannot be determined for the data service at '"
-                            + file.getAbsolutePath() + "'");
-        }
-        return serviceName;
-    }
+		try (FileInputStream fis = new FileInputStream(file.getAbsoluteFile())) {
```

Comment:
Indentation issue.. Indentation should be set to four spaces..

Reply:
Fixed. Please refer latest commit.